types/redux-form/lib/reduxForm.d.ts

```diff
@@ -18,7 +18,7 @@ import {
 } from "../index";
 
 export type FormSubmitHandler<FormData = {}, P = {}> =
-    (values: Partial<FormData>, dispatch: Dispatch<any>, props: P) => void | FormErrors<FormData> | Promise<any>;
+    (values: FormData, dispatch: Dispatch<any>, props: P) => void | FormErrors<FormData> | Promise<any>;
```

Comment:
I was rather confused when I looked at this because `FormData` is an actual DOM API constructor/instance and it makes no sense to have a `Partial<FormData>`. Naming it `TFormData` might make it less confusing, but that's not the point of the PR 🤷‍♀️